texar/torch/data/tokenizers/roberta_tokenizer.py

```diff
@@ -24,8 +24,7 @@
     'RoBERTaTokenizer',
 ]
 
-_GPT2_PATH = "https://dl.fbaipublicfiles.com/fairseq/gpt2_bpe/"
-_CHECKPOINT_FILES = ["encoder.json", "vocab.bpe"]
+_ROBERTA_PATH = "https://s3.amazonaws.com/models.huggingface.co/bert/"
```

Comment:
Can we host the checkpoints by ourselves, on the asyml google drive?

Reply:
We can. Do we currently have asyml google drive?

Reply:
> > `input_ids, _ = tokenizer.encode_text('Hello world!', max_seq_length=5)`
> > is `max_seq_length` necessary here? what's the result in this case without setting `max_seq_length`?
> 
> `max_seq_length` is used here just to show the same result between ours and theirs. If `max_seq_length` is not specified, it will be `[0, 31414, 232, 328, 2, 0, 0, 0, ..., 0]`(zero-pad up to the maximum sequence length).

If the user doesn't want padding, it's difficult for them to know the correct seq length just as in this case ('Hello world!' needs seq_length 5).. Can we add an argument (or allow a special value of `max_seq_length`) so that the user can get the encoded text without padding (and without need to specify `max-seq-length` explicitly)?